db_test.go

```diff
@@ -425,6 +425,40 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+// BenchmarkCreate benchmarks the creation of a record
+func BenchmarkCreate(b *testing.B) {
+	s := new(SurrealDBTestSuite)
+	s.SetT(&testing.T{})
+	s.SetupSuite()
+	user := &testUser{
+		Username: "tobi",
+		Password: "1234",
+	}
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		s.db.Create(fmt.Sprintf("users:%d", i), user)
```

Comment:
You should probably move string interpolation outside the benchmark. Have a slice of pre-made strings, perhaps.

Reply:
done